runtime/src/bank.rs

```diff
@@ -2985,6 +2985,34 @@ impl Bank {
             .collect()
     }
 
+    pub fn weight_packet(&self, packet: &Packet) -> u64 {
+        if !packet.meta.is_simple_vote_tx {
+            return 0;
+        }
+
+        fn vote_address_from_packet(packet: &Packet) -> Option<Pubkey> {
+            let tx: VersionedTransaction =
+                limited_deserialize(&packet.data[0..packet.meta.size]).ok()?;
```

Comment:
Possible to get account pubkey by continue counting packet offsets instead of calling deserialization.

Reply:
Yeah agreed, I think this part could have a negative impact on perf. I'm not sure if our perf tests will see hit it though since they don't process many vote tx's. We'll probably have to bake on v1.7